index.html

```diff
@@ -43,6 +43,254 @@
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                    </div> 
+                </div>
+                <div class="button_bg">
+                    <div class="btn">
+                        <a href="#"> See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="posts">
+                <div class="p_title">
+                    <h1>Profesional Art <br>Printing Data</h1>
+                    <p>
+                        A daily selection of privately 
+                        <br>personalized reads; no accounts or
+                        <br>sign-ups required. Has been the 
+                        <br>industry's standard.
+                    </p>
+                    <div class="p_p">
+                        <ul>
+                            <li><a href="#">HTML</a></li>
+                            <li><a href="#">Bootstrap</a></li>
+                            <li><a href="#">Ruby</a></li>
+                        </ul>
+                    </div> 
+                </div>
+                <div class="button_bg">
+                    <div class="btn">
+                        <a href="#"> See Projects</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="text-about">
+            <h1>About me</h1>
+            <p>
+                Hello I’m a software developer! I can help
+                <br>you build a product , feature or website
+                <br>Look through some of my work and
+                <br>experience! If you like what you see and
+                <br>have a project you need coded, don’t
+                <br>hesitate to contact me.
+            </p>
+            <div class="abt-btn">   
+                <a href="#">Get my resume</a> 
+            </div>
+         </div>   
+         <hr />
+         
+        <ul class="lang-menu">
+            <li class="langs-1">
+                <h3>Languages</h3>
+                <ul class="lang-list" >
+                    <li>
+                        <h4>JavaScript</h4>
+                    </li>
+                    <li>
+                        <h4>Ruby</h4>
+                    </li>
+                    <li>
+                        <h4>HTML</h4>
+                    </li>
+                    <li>
+                        <h4>CSS</h4>
+                    </li>
+                    <li>
+                        <h4>PHP</h4>
+                    </li>
```

Comment:
Kindly, I suggest you use the `p` tag instead of `h4`, also make sure only the sub-heading is bold.
According to the Figma design, I suggest you fix it to make your project look more professional.

> ## The Figma design
>
><img width="250" alt="Screenshot 2023-06-03 at 12 00 33 PM" src="https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/91237525/611bbf9a-a15a-4a67-af46-cc6e8adc1a67">
>
> ## Your Design
>
><img width="368" alt="Screenshot 2023-06-03 at 12 01 18 PM" src="https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/91237525/bb4b01e5-0c2e-4979-81bb-50e6429cc210">